sql/populateTables.sql

```diff
@@ -0,0 +1,15 @@
+INSERT INTO flashcard (front, back) VALUES ('pear', 'is a green fruit');
+INSERT INTO flashcard (front, back) VALUES ('banana', 'is a yellow fruit');
+INSERT INTO flashcard (front, back) VALUES ('tomato', 'is a red fruit');
+INSERT INTO flashcard (front, back) VALUES ('blood', 'is red');
+INSERT INTO flashcard (front, back) VALUES ('mars', 'is a red planet');
```

Comment:
These could be a multiple values insert statement as opposed to multiple single lines